waterloop-site/src/components/SustainableTech/Teams/TeamsCarousel.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
```

Comment:
On my system, it seems like this file hasn't been formatted. Can you make sure that all the files you're committing are formatted with Prettier?

Reply:
Yeah fixed it, it because I did this section on another computer and I don't think I had Prettier enabled on VS Code. But it should be formatted correctly now that I'm back to my original computer.